src/flexbox.rs

```diff
@@ -106,17 +106,17 @@ struct AlgoConstants {
     inner_container_size: Size<f32>,
 }
 
-impl Taffy {
+impl<T> Taffy<T> {
     /// Computes the layout of [`Taffy`] according to the flexbox algorithm
-    pub(crate) fn compute(&mut self, root: Node, size: Size<Option<f32>>) {
+    pub(crate) fn compute(&mut self, root: Node, size: Size<Option<f32>>, measure: &impl Measure<T>) {
```

Comment:
A reference to a type that implements a trait definitely feels like a Rustier design.

Reply:
isn't this just a syntax sugar for it? https://doc.rust-lang.org/stable/rust-by-example/trait/impl_trait.html#as-an-argument-type